src/scss/grommet-core/_objects.layer.scss

```diff
@@ -111,6 +111,14 @@
       max-height: calc(100vh - #{double($inuit-base-spacing-unit)});
       max-width: calc(100vw - #{double($inuit-base-spacing-unit)});
       transform: translate(50%, 50%);
+
```

Comment:
Thanks for your contribution, but I guess this problem is unrelated with the align center aspect of the layer.

I believe the reason for the problem is this line https://github.com/grommet/grommet/blob/master/src/scss/grommet-core/_objects.layer.scss#L61

I'm currently investigating it more.